tesseract_command_language/test/utils_test.cpp

```diff
@@ -480,6 +481,34 @@ TEST(TesseractCommandLanguageUtilsUnit, generateSkeletonSeed)
   }
 }
 
+TEST(TesseractCommandLanguageUtilsUnit, toDelimitedFile)
+{
+  CompositeInstruction composite;
+  composite.setDescription("To Delimited File: Composite");
+
+  std::vector<std::string> joint_names = { "1", "2", "3" };
+  Eigen::VectorXd values = Eigen::VectorXd::Constant(3, 5);
+  composite.push_back(MoveInstruction(JointWaypoint(joint_names, values), MoveInstructionType::FREESPACE));
+  values = Eigen::VectorXd::Constant(3, 10);
+  composite.push_back(MoveInstruction(JointWaypoint(joint_names, values), MoveInstructionType::FREESPACE));
+  values = Eigen::VectorXd::Constant(3, 15);
+  composite.push_back(MoveInstruction(JointWaypoint(joint_names, values), MoveInstructionType::FREESPACE));
+
+  std::string path = tesseract_common::getTempPath() + "to_delimited_file.csv";
+  toDelimitedFile(composite, path);
+
+  std::string check_string = "";
+
+  //  std::string outputstring;
+  //  std::string buffer;
+  //  std::ifstream input(path);
+
+  //  while (std::getline(input, buffer))
+  //      outputstring += (buffer + '\n');
+
+  //  EXPECT_EQ(outputstring, check_string);
```

Comment:
Commented code here too. I wouldn't let the delimited file code hold this up though if that is a problem.

Reply:
Forget to finish. I will fix it.

Reply:
fixed.